src/arguments.cpp

```diff
@@ -55,8 +56,31 @@ const char* Arguments::parse(char* args) {
             _action = ACTION_STOP;
         } else if (strcmp(arg, "status") == 0) {
             _action = ACTION_STATUS;
-        } else if (strcmp(arg, "cpu") == 0) {
+        } else if (strncmp(arg, "cpu", 3) == 0) {
```

Comment:
Does `-m cpu:cache-misses` syntax have any advantages over just `-m cache-misses`? Any objects against extending `enum Mode` with all these events?

Reply:
I don't mind, I was treating 'cpu' as the gateway to perf profiling, so in that sense 'perf:cpu' and 'perf:cache-misses' made sense to me. But modes work equally well, or following the perf notation we could add an option: '-e cache-misses'